### DISCUSSION TITLE
SYNTAX ERROR: Only i32 is supported for now

### DISCUSSION THREAD

**Discussion by yaa-ali** [carbon-language/carbon-lang#2429] [General]
Hello, I encountered this problem surprisingly
> SYNTAX ERROR: ./variables.carbon:6: Only i32 is
> supported for now: f32
```
package JetBlack api;

fn Main() -> i32 {
    var policy:i32;

    var f :f32 =5.6; // this line 
    return 0;
}
```
Why?

**Comment by jonmeow** [comment]
It's just a matter of what we've implemented so far; the implementation is incomplete as we were preparing to implement the types as Carbon library types versus C++ code (as is done for i32). See #767.

**Reply by yaa-ali** [reply]
*Thank you*
